mysql-test/suite/rocksdb/t/tmp_table.test

```diff
@@ -21,6 +21,8 @@ select table_schema, substring(table_name, 1, 4), index_name from information_sc
 disconnect con1;
 
 connection default;
+let $count_sessions= 1;
```

Comment:
May be canonical `--source include/count_sessions.inc` at the beginning of this test instead of `let $count_sessions= 1;`?

Reply:
Done